dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -237,7 +256,87 @@
         typeMap.put("Geolocation", TypeUtil.createObjectType("Geolocation",
                 geoFields));
 
+        // Layout type
+        final Map<String, TypeFetcher> layoutFields = new HashMap<>();
+        layoutFields.put("width", new TypeFetcher(GraphQLLong,
+                new PropertyDataFetcher<TemplateLayout>("width")));
+        layoutFields.put("title", new TypeFetcher(GraphQLString,
+                new PropertyDataFetcher<TemplateLayout>("title")));
+        layoutFields.put("header", new TypeFetcher(GraphQLBoolean,
+                new PropertyDataFetcher<TemplateLayout>("header")));
+        layoutFields.put("footer", new TypeFetcher(GraphQLBoolean,
+                new PropertyDataFetcher<TemplateLayout>("footer")));
+        layoutFields.put("body", new TypeFetcher(GraphQLTypeReference.typeRef("Body"),
+                new PropertyDataFetcher<TemplateLayout>("body")));
+        layoutFields.put("sidebar", new TypeFetcher(GraphQLTypeReference.typeRef("Sidebar"),
+                new PropertyDataFetcher<TemplateLayout>("sidebar")));
+
+        typeMap.put("Layout", TypeUtil.createObjectType("Layout",
+                layoutFields));
+
+        // Sidebar type
+        final Map<String, TypeFetcher> sidebarFields = new HashMap<>();
+        sidebarFields.put("widthPercent", new TypeFetcher(GraphQLLong,
```

Comment:
Codacy found an issue: [The String literal "widthPercent" appears 4 times in this file; the first occurrence is on line 279](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6084048)